app/config/AppConfig.scala

```diff
@@ -63,20 +63,9 @@ class AppConfig @Inject()(val config: Configuration, servicesConfig: ServicesCon
     else etmpBase + "/"
   }
 
-  def getPenaltyDetailsUrl: String = {
-    if(!isEnabled(CallAPI1812ETMP)) stubBase + "/penalties-stub/penalty/details/VATC/VRN/"
-    else etmpBase + "/penalty/details/VATC/VRN/"
-  }
-
-  def getFinancialDetailsUrl: String = {
-    if(!isEnabled(CallAPI1811ETMP)) stubBase + "/penalties-stub/penalty/financial-data/"
-    else etmpBase + "/penalty/financial-data"
-  }
```

Comment:
We still want to call the stub or EIS/ETMP dependent on the feature switch value